indico/modules/attachments/forms.py

```diff
@@ -104,3 +104,7 @@ class AttachmentPackageForm(IndicoForm):
                                                  description=_('Include attachments from selected sessions'))
     contributions = IndicoSelectMultipleCheckboxField(_('Contributions'),
                                                       description=_('Include attachments from selected contributions'))
+    contributions_schedule_dates = IndicoSelectMultipleCheckboxField(_('Contributions scheduled on'),
```

Comment:
"Dates" might be a better label for this. I guess it doesn't only apply to contributions but also e.g. sessions